status/versions.go

```diff
@@ -267,7 +267,7 @@ func istioVersion() (*ExternalServiceInfo, error) {
 		Put(MeshVersion, meshVersion)
 		Put(MeshName, meshName)
 		AddWarningMessages(compatibleWarnings)
-		return nil, fmt.Errorf(compatibleWarnings)
+		//return nil, fmt.Errorf(compatibleWarnings)
```

Comment:
remove this

Reply:
well, I left that in there for a reason. This seems to me to be a big change.  This is what I was talking about when I said, `This makes a change to status/version.go whereby this function no longer returns an error if the istio cluster type/version is unknown. We need to test this thoroughly to make sure this doesn't break something.`

I left that commented out because in the future I predicate a 50/50 chance we will want to resurrect that line quickly to fix a bug we can't see right now.

Reply:
I added a comment to the comment in commit https://github.com/kiali/kiali/pull/5721/commits/e12b9b166b5d43ce2666c77657d4ea259e5feff4 that points this out